index.html

```diff
@@ -190,9 +190,26 @@ <h2 class="title">
    
         </section>
 
+        <section class="contact-me">
+            
+        <div class="text-contact">  
+            <h3 id="contact_title">
+                Contact me
+            </h3>
+           
+        </div>
+           <p id="form_text">
+                If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!
+            </p>
 
-    
- 
+        <form action='https://formspree.io/f/xlekzqzw' method="POST" id="form" >
+            <input type="name" name="name" required placeholder="Lorem Ipsum" maxlength="30">
```

Comment:
- [ ] Kindly consider changing `type` from name to `text`. also consider writing `Name` in placeholder instead of `Lorem Ipsum` because we should have appropriate type and placeholder text in input.